ols/app/endpoints/feedback.py

```diff
@@ -6,10 +6,11 @@
 from fastapi import APIRouter, Depends
 
 from ols.app.models.models import FeedbackRequest, FeedbackResponse
-from ols.utils.auth_dependency import auth_dependency
+from ols.utils.auth_dependency import AuthDependency
 
 logger = logging.getLogger(__name__)
 router = APIRouter(prefix="/feedback", tags=["feedback"])
+auth_dependency = AuthDependency(virtual_path="/ols-metrics-access")
```

Comment:
should be /ols-access since i think we've said ols access should be the same as feedback access (the case where it's not is listing feedback and deleting feedback, but since we expect to drop those apis entirely, we don't need to concern ourselves with them)

(definitely shouldn't be /ols-metrics-access regardless, guessing cut+paste error)